citre-global.el

```diff
@@ -118,8 +118,11 @@ START-FILE."
       (_ (error "Invalid MODE")))
     (when case-fold (push "--ignore-case" cmd))
     ;; Global doesn't know how to expand "~", so we need to expand START-FILE.
-    (when start-file (push (concat "--nearness=" (expand-file-name start-file))
-                           cmd))
+    (when start-file (let ((file (if (tramp-tramp-file-p start-file)
+                                     (tramp-file-name-localname (tramp-dissect-file-name (tramp-handle-expand-file-name start-file)))
+                                   (expand-file-name start-file))))
+                         (push (concat "--nearness=" file)
+                            cmd)))
```

Comment:
Let's change this block to:

```elisp
    (when start-file
      (let ((file (if (file-remote-p start-file)
                      (tramp-file-name-localname
                       (tramp-dissect-file-name
                        (tramp-handle-expand-file-name start-file)))
                    (expand-file-name start-file))))
        (push (concat "--nearness=" file)
              cmd)))
```

so it fits the 79 column limit. Also, use `file-remote-p` instead of `tramp-tramp-file-p` so we don't have to `(require 'tramp)` in this file.

Reply:
done